monitor/monitor.go

```diff
@@ -125,7 +132,7 @@ func (m *EthMonitor) monitorBlockHead() {
 }
 
 func (m *EthMonitor) monitorInitializeCandidate() {
-	_, err := m.ms.Monitor(string(InitializeCandidate), m.guardContract, m.blkNum, nil, false, func(cb watcher.CallbackID, eLog ethtypes.Log) {
+	_, err := m.ms.Monitor(string(InitializeCandidate), m.dposContract, m.sgnContract, m.blkNum, nil, false, func(cb watcher.CallbackID, eLog ethtypes.Log) {
```

Comment:
Why adding another argument to monitor function? It's obvious that Monitor() func only takes one contract as arg, and add arg without changing the func itself won't compile for sure.

Reply:
Thanks. I will fix it soon.